sqlglot/optimizer/scope.py

```diff
@@ -353,10 +357,27 @@ def selects(self):
         Returns:
             list[exp.Expression]: expressions
         """
-        if isinstance(self.expression, exp.Union):
```

Comment:
why'd you get rid of the union?

Reply:
Because `Union.selects` (and `Union.named_selects`) are already implemented like this:

https://github.com/tobymao/sqlglot/blob/3b215adc413772bc1af46a67e2410b38ad8872a2/sqlglot/expressions.py#L2401-L2402